### ISSUE TITLE
Exit menu entry

### ISSUE THREAD

**Issue by dbojan** [osmandapp/Osmand#7828]
Hi.
-Please add "Exit" menu entry.
Reason: Some users do not know how to stop the program navigation. 

-When user preses "back" buton on the device to exit the program, please show prompt "Do you wish to close the program?" Yes/No
Reason: I find it very usefull on some other programs, like Magic Earth.

Thanks.

**Comment by sonora** [comment]
Mapping apps should not need this, we should let the OS decide when to kill an Activity.

**Comment by SviMik** [comment]
> When user preses "back" buton on the device to exit the program, please show prompt "Do you wish to close the program?" Yes/No

I agree it would be useful. Waze has this feature too and I like it.

**Comment by sonora** [comment]
I just notice the issue may be talking about stopping a navigation, not exiting the app  (which is not the same...) 

In deed, I have also encountered users not knowing how to interrupt a navigation, there may be a point here.

**Comment by dbojan** [comment]
Actually it is both.
I set up navigation for another person, and she did not know how to stop the program (from announcing the turns ... etc).

For myselft, when I exit the program, I usually have drop notification drawer, and click on stop navigation, which is an extra step..
